internal/launcher/launcher.go

```diff
@@ -300,7 +300,7 @@ func (l *launcher) reconfigureWait(timeout time.Duration) (*config.C, error) {
 					l.log.Errorf("Config update validation failed: %v", err)
 					healthErr := &BeaterUnhealthyError{}
 					if errors.As(err, healthErr) {
-						l.beat.Manager.UpdateStatus(management.Degraded, healthErr.Error())
+						l.beat.Manager.UpdateStatus(status.Degraded, healthErr.Error())
```

Comment:
This change is needed since this: https://github.com/elastic/beats/commit/38a678be92e4588e194bcbecad38634b62c92b84#diff-6a75dda9050c11ec2774e66f1cbfa17918dd3558f3726dfee6c6cbfe3796914a